pkg/commands/artifact/run.go

```diff
@@ -215,7 +216,7 @@ func (r *runner) scanArtifact(ctx context.Context, opts flag.Options) (types.Rep
 // Run performs artifact scanning
 func Run(ctx context.Context, opts flag.Options, targetKind TargetKind) (err error) {
 	if !opts.Quiet {
-		outputhandler.StdErrLogger().Msg("Loading rules")
+		output.StdErrLogger().Msg("Loading rules")
```

Comment:
Curious why we can't keep `outputhandler` here? Doesn't it mean we are importing `pkg/util/output` twice?